LadybugTools_Engine/Python/src/ladybugtools_toolkit/wind.py

```diff
@@ -1675,7 +1675,7 @@ def plot_windmatrix(
         if other_data is None:
             other_data = self.ws
 
-        title = self.source
+        title = self.source if self.source is not None else "title"
         title += f'\n{kwargs.pop("title", None)}'
         ax.set_title(textwrap.fill(f"{self.source}", 75))
```

Comment:
```suggestion
        ax.set_title(textwrap.fill(f"{title}", 75))
```

noticed this as well